Cargo.toml

```
@@ -1,7 +1,7 @@
 [package]
 name = "pyoxipng"
-version = "6.0.0"
-edition = "2021"
+version = "7.0.0"
+edition = "2022
```

Missing quote here, also there is no rust 2021

```suggestion
edition = "2021"
```